koku/api/report/aws/query_handler.py

```diff
@@ -318,8 +318,11 @@ def _format_query_response(self):
             (Dict): Dictionary response of query params, data, and total
 
         """
-        output = self._initialize_response_output(self.parameters)
 
+        if self._report_type == "ec2_compute":
+            self.query_data = self._format_ec2_response(copy.deepcopy(self.query_data))
```

Comment:
(question) what do we gain by using deepcopy here? format_ec2_response is modifying the copy, but then we just overwrite self.query_data anyway. I think from a memory perspective, we shouldn't use deepcopy unless it's really necessary.

Reply:
Deepcopy is not necessary, updated to just overwrite `self.query_data`